apps/landing-page/pages/community.mdx

```diff
@@ -20,7 +20,7 @@ export function CommunityHeader() {
 
 ## Join the community
 
-Start by joining our community. We have a [janus-idp](https://join.slack.com/t/janus-idp/shared_invite/zt-1pxtehxom-fCFtF9rRe3vFqUiFFeAkmg) Slack workspace that you can join and meet other Janus users. In all our communications, on whatever platform, we expect everyone to follow a code of contact.
+Start by joining our community. We have a [janus-idp](https://janus-idp.slack.com/archives/C04EDTPJRK5) Slack workspace that you can join and meet other Janus users. In all our communications, on whatever platform, we expect everyone to follow a code of contact.
```

Comment:
https://join.slack.com/t/janus-idp/shared_invite/zt-1pxtehxom-fCFtF9rRe3vFqUiFFeAkmg is the proper link

Reply:
Updated!